barretenberg/cpp/src/barretenberg/client_ivc/mock_circuit_producer.hpp

```diff
@@ -209,10 +209,30 @@ class PrivateFunctionExecutionMockCircuitProducer {
         return { circuit, get_verification_key(circuit, ivc.trace_settings) };
     }
 
+    void accumulate_next_circuit(ClientIVC& ivc, TestSettings settings = {})
```

Comment:
Seems like this method should be called something like `construct_and_accumulate_next_circuit()`